doc/using/InvokingGHDL.rst

```diff
@@ -624,64 +627,64 @@ have been split for readability):
 
-With some analyser, the explicit declaration has priority over the implicit declaration, and this design can be analyzed without error.  However, this is not the rule given by the VHDL LRM, and since GHDL follows these rules,
+With some analysers, the explicit declaration has priority over the implicit declaration, and this design can be analyzed without error.  However, this is not the rule given by the VHDL LRM, and since GHDL follows these rules,
 it emits an error.
 
-You can force GHDL to use this rule with the *-fexplicit* option (see :ref:`GHDL_options` for further details). However it is easy to fix this error, by using a selected name:
+You can force GHDL to use this rule with the *-fexplicit* option (see ':ref:`GHDL:options`' for further details). However it is easy to fix this error, by using a selected name:
 
 .. code-block:: VHDL
 
-  library ieee;
-  use ieee.std_logic_unsigned.all;
-
-  architecture fixed_bad of counter
-  is
-     signal v : std_logic_vector (3 downto 0);
-  begin
-     process (ck, rst)
-     begin
-       if rst = '1' then
-          v <= x"0";
-       elsif rising_edge (ck) then
-          if ieee.std_logic_unsigned."=" (v, "1010") then
-             v <= x"0";
-          else
-             v <= v + 1;
-          end if;
-       end if;
-     end process;
-
-     val <= v;
-  end fixed_bad;
-
-It is better to only use the standard packages defined by IEEE, which provides the same functionalities:
+   library ieee;
+   use ieee.std_logic_unsigned.all;
+
+   architecture fixed_bad of counter
+   is
+      signal v : std_logic_vector (3 downto 0);
+   begin
+      process (ck, rst)
+      begin
+         if rst = '1' then
+            v <= x"0";
+         elsif rising_edge (ck) then
+            if ieee.std_logic_unsigned."=" (v, "1010") then
+               v <= x"0";
+            else
+               v <= v + 1;
+            end if;
+         end if;
+      end process;
+
+      val <= v;
+   end fixed_bad;
+
+It is better to only use the standard packages defined by IEEE, which provide the same functionalities:
 
 .. code-block:: VHDL
 
-  library ieee;
-  use ieee.numeric_std.all;
-
-  architecture good of counter
-  is
-     signal v : unsigned (3 downto 0);
-  begin
-     process (ck, rst)
-     begin
-       if rst = '1' then
-          v <= x"0";
-       elsif rising_edge (ck) then
-          if v = "1010" then
-             v <= x"0";
-          else
-             v <= v + 1;
-          end if;
-       end if;
-     end process;
-
-     val <= std_logic_vector (v);
-  end good;
+   library ieee;
+   use ieee.numeric_std.all;
+
+   architecture good of counter
+   is
+      signal v : unsigned (3 downto 0);
+   begin
+      process (ck, rst)
+      begin
+         if rst = '1' then
+            v <= x"0";
+         elsif rising_edge (ck) then
+            if v = "1010" then
```

Comment:
Why has indentation changed?

Reply:
The first example of a code block I found was in the [Quick Start Guide](https://ghdl.readthedocs.io/en/latest/using/QuickStartGuide.html). It used a 3-space indent. Therefore, I standardized all other documents to use a 3-space indent inside code block examples.